src/Avalonia.Controls/DragDrop/IDragData.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace Avalonia.Controls.DragDrop
+{
+    public interface IDragData
```

Comment:
Might be worth to rename to [IDataObject](https://docs.microsoft.com/en-us/dotnet/api/system.windows.idataobject?view=netframework-4.7.1). We should probably also use it from our clipboard later.

Reply:
Done.